docs/user-guide.md

```diff
@@ -24,34 +25,50 @@ This guide provides detailed instructions on how to configure your Liberty proje
 
-To resolve this conflict, define `java.jdt.ls.java.home` and `xml.java.home` in your Visual Studio Code [settings.json](https://code.visualstudio.com/docs/getstarted/settings) file to point Liberty Tools to your Java 17 or later runtime.
+For both Liberty Config Language Server and Eclipse Language Server for Jakarta EE, Liberty Tools for Visual Studio Code will check for the Java versions in the following order:
+1. The [embedded JRE](https://github.com/redhat-developer/vscode-java#java-tooling-jdk) included by [Language Support for Java(TM) by Red Hat](https://marketplace.visualstudio.com/items?itemName=redhat.java). 
+  - By default, Liberty Tools installs the latest version of the rehat.java plugin, which contains an embedded JRE higher than Java 17 and therefore no configuration is required. However, if using an older version of the redhat.java plugin or using the universal version without the embedded JRE, use `java.jdt.ls.java.home` to specify your new JDK path.
+2. `JDK_HOME` or `JAVA_HOME` as system environment variables. (Note: if both `JDK_HOME` and `JAVA_HOME` are set, `JDK_HOME` will take precedence)
+
+For the XML Language Server, or LemMinX, Liberty Tools for Visual Studio Code will check for the Java versions in the following order:
+1. `xml.java.home` in settings.json
+2. `JDK_HOME` or `JAVA_HOME` as system environment variables. (Note: if both `JDK_HOME` and `JAVA_HOME` are set, `JDK_HOME` will take precedence)
 
 ![settings.json example](/docs/screenshots/settings.json%20path%20example.png)
 
-Liberty Tools for Visual Studio Code checks for the Java runtime in the following order of precedence:
-1. The [embedded JRE](https://github.com/redhat-developer/vscode-java#java-tooling-jdk) included by [redhat.java plugin](https://marketplace.visualstudio.com/items?itemName=redhat.java)
-2. `java.jdt.ls.java.home` in settings.json
-3. `java.home` in settings.json. Note: This setting is [deprecated!](https://github.com/redhat-developer/vscode-java#supported-vs-code-settings), use 'java.jdt.ls.java.home' instead. 
-4. `JDK_HOME`
-5. `JAVA_HOME`
+### External settings
+
+The following settings, which are provided by external extensions, are honoured when you run dev mode commands.
+
+| Setting | Description | Provided By |
+| --- | --- | --- |
+| `maven.executable.path` | Maven commands executed by dev mode will honour this setting. When this value is empty, it tries to use `mvn` or `mvnw` according to the value of `maven.executable.preferMavenWrapper`. | [Maven for Java extension](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-maven) |
+| `maven.executable.preferMavenWrapper` | Maven commands executed by dev mode will honour this setting. If true, it tries to use `mvnw` if a Maven wrapper file can be found. Otherwise it will use `mvn`. | [Maven for Java extension](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-maven) |
```

Comment:
```suggestion
| `maven.executable.preferMavenWrapper` | Maven commands executed by dev mode honour this setting. If this setting set to `true`, dev mode tries to use `mvnw` if a Maven wrapper file can be found. Otherwise, it uses `mvn`. | [Maven for Java extension](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-maven) |
```